package.json

```diff
@@ -232,6 +251,14 @@
                                 },
                                 "default": []
                             },
+                            "problemMatchers": {
```

Comment:
I expected that we have the same schema as the vscode problem matchers for tasks here. We want to support more than just named problem matchers. Users should be able to specify their own regex patterns too.

Reply:
I will start a conversation with the VSCode team (I didn't open a GitHub issue yet about API to create a task with a problem matcher based on it's complete schema versus just a string name, will do today). Since currently it is not supported, even if they approve the request it will be part of the next VSCode release, so in the mean time we will have to accept this limitation.

A global problem matcher defined in .vscode/tasks.json does not have a name or label. But we can do something like simply read it, assign it a name that we use when we create the task. Let's discuss in the morning meeting.